lib/utils/storage.ts

```diff
@@ -0,0 +1,61 @@
+import * as azure from "azure-storage";
```

Comment:
since this is specifically for Azure Storage, better call the file `azure_storage.ts`

Reply:
ok